src/Components/Main/WatchedBox/WatchedSummary.jsx

```diff
@@ -1,4 +1,7 @@
-export const WatchedSummary = ({ watched, average }) => {
+const average = (arr) =>
+  arr.reduce((acc, cur, i, arr) => acc + cur / arr.length, 0);
```

Comment:
provide understandable name to arguments

Reply:
done